README.md

```diff
@@ -75,16 +72,60 @@ By default, kola uses the `qemu` platform with the most recently built image
 
 #### kola run
 
-In order to provide internet access to QEMU instances; it's required to enable ipv4 forwarding on the host machine:
+##### Getting started with QEMU
 
+The easiest way to get started with `kola` is to run a `qemu` test.
+
+***requirements***:
+ - IPv4 forwarding (to provide internet access to the instance): `sudo sysctl -w net.ipv4.ip_forward=1`
+ - `dnsmasq`, `go` and `iptables` installed and present in the `$PATH`
+ - `qemu-system-x86_64` and / or `qemu-system-aarch64` to respectively tests `amd64` and / or `arm64`
+
+From the pulled sources, `kola` and `kolet` must be compiled:
+
+```shell
+git clone https://github.com/kinvolk/mantle/
+cd mantle
+./buid kola kolet
+```
+
+Finally, a Flatcar image must be available on the system:
+  - from a locally [built](https://kinvolk.io/docs/flatcar-container-linux/latest/reference/developer-guides/sdk-modifying-flatcar/) image
+  - from an official [release](https://kinvolk.io/flatcar-container-linux/releases/)
+
+###### Run tests for AMD64
+
+Example with the latest `alpha` release:
 ```shell
-sudo sysctl -w net.ipv4.ip_forward=1
+wget https://alpha.release.flatcar-linux.net/amd64-usr/current/flatcar_production_qemu_image.img.bz2
+wget https://alpha.release.flatcar-linux.net/amd64-usr/current/flatcar_production_qemu_image.img.bz2.sig
+gpg --verify ./flatcar_production_qemu_image.img.bz2.sig
+bzip -dk ./flatcar_production_qemu_image.img.bz2
+sudo ./bin/kola run --key ${HOME}/.ssd/id_rsa.pub -k -b cl -p qemu --qemu-image ./flatcar_production_qemu_image.img cl.locksmith.cluster
 ```
 
-The run command invokes the main kola test harness. It
-runs any tests whose registered names matches a glob pattern.
+###### Run tests for ARM64
+Example with the latest `alpha` release:
+```shell
+wget https://alpha.release.flatcar-linux.net/arm64-usr/current/flatcar_production_image.bin.bz2
+wget https://alpha.release.flatcar-linux.net/arm64-usr/current/flatcar_production_image.bin.bz2.sig
+gpg --verify ./flatcar_production_image.bin.bz2.sig
+bzip2 -dk ./flatcar_production_image.bin.bz2.sig
+
+wget https://alpha.release.flatcar-linux.net/arm64-usr/current/flatcar_production_qemu_uefi_efi_code.fd
+wget https://alpha.release.flatcar-linux.net/arm64-usr/current/flatcar_production_qemu_uefi_efi_code.fd.sig
+gpg --verify ./flatcar_production_qemu_uefi_efi_code.fd.sig
 
-`kola run <glob pattern>`
+sudo ./bin/kola run --board arm64-usr --key ${HOME}/.ssh/id_rsa.pub -k -b cl -p qemu --qemu-image ./flatcar_production_image.bin --qemu-bios=./flatcar_production_qemu_uefi_efi_code.fd cl.etcd-member.discovery
+```
+
+_Note for both architectures_:
+- `sudo` is required because we need to create some `iptables` rules to provide QEMU Internet access
+- using `--key`, it's possible to SSH into the created instances - PID identification of the `qemu` instance is required:
```

Comment:
Maybe we can also recommend to run with `--remove=false -d` to see the QEMU PIDs and have the VM still running, and how to kill the QEMU and dnsmasq instances afterwards.

Reply:
Good idea !

Reply:
Should works by adding `2> >(tee pidlog >&2) ` and then `sudo kill $(grep 'manual cleanup needed' pidlog | cut -d : -f 5)` to clean up